core/PhysiCell_standard_models.cpp

```diff
@@ -1231,13 +1259,49 @@ void standard_cell_cell_interactions( Cell* pCell, Phenotype& phenotype, double
 			double attack_ij = phenotype.cell_interactions.attack_rate(type_name); 
 			double immunogenicity_ji = pTarget->phenotype.cell_interactions.immunogenicity(pCell->type_name); 
 
+			// probability of STARTING an attack 
 			probability = attack_ij * immunogenicity_ji * dt; 
-			
-			if( UniformRandom() < probability && attacked == false ) 
+			if( attacked == false && pCell->phenotype.cell_interactions.pAttackTarget == NULL )
```

Comment:
@MathCancer I think we may want to initialize `attacked` above as 
```
bool attacked = pCell->phenotype.cell_interactions.pAttackTarget != NULL
```
Otherwise, here we are allowing a cell that is continuing to attack from a previously initiated attack to start attacking a new target. that's confusing so here's what could happen:

1. At $t=t_1$, T cell $T$ initiates an attack against cancer cell $C_1$. During $t=t_1$ update, $T$ cannot start a separate attack against any other cell
2. At time $t=t_2>t_1$, $T$ is still attacking $C_1$. It chooses to initiate an attack against a different cancer cell, $C_2$ so that now it is attacking two cells.
I think some bad things could happen, e.g., $T$ is now targeting $C_2$ but still spring-attached to $C_1$.